Create accts.transactions.mdx (#145)
comcode-org/hackmud_wiki: docs/scripting/trust scripts/accts.transactions.mdx

---
title: accts.transactions
---

## Description

accts.transactions returns a log of the user's transactions.

## Syntax

> On the Command Line Interface:

Reviewer: ```suggestion
### CLI
```

Brings inline with template

```
accts.transactions
```

> As a subscript:

Reviewer: ```suggestion
### Script
```

Brings inline with template

```
#hs.accts.transactions()
```

### Parameters

#### count

The 'count' argument allows a user to specify the number of transactions to be returned.

#### start

> The 'start' argument can be used in combination with 'count', or on it's own, to offset the beginning of the list to the desired position.

Reviewer: ```suggestion
The 'start' argument offsets the beginning of the list of transactions to the desired position. Can be used in combination with 'count'.
```

Reviewer: Maybe less confusing wording?

#### to

> A user may filter transactions that are sent to a specific username.

Reviewer: ```suggestion
The 'to' argument filters transactions that are sent to a specific username.
```

#### from

> A user may filter transactions that are sent from a specific username.

Reviewer: ```suggestion
The 'from' argument filters transactions that are sent from a specific username.
```

#### script

> A user may filter transactions by the specific script which sent them.

Reviewer: ```suggestion
The 'script' argument filters transactions by the specific script which sent them.
```
